server/src/main/java/com/google/sps/utils/WriteAd.java

```diff
@@ -0,0 +1,36 @@
+package com.google.sps.utils;
+import com.google.api.core.ApiFuture;
+import com.google.auth.oauth2.GoogleCredentials;
+import com.google.cloud.firestore.DocumentReference;
+import com.google.cloud.firestore.Firestore;
+import com.google.cloud.firestore.WriteResult;
+import com.google.firebase.FirebaseApp;
+import com.google.firebase.FirebaseOptions;
+import com.google.firebase.cloud.FirestoreClient;
+import java.io.FileInputStream;
+import java.io.InputStream;
+
+/* 
+ * Description: Utility class that writes as objects to Firestore. 
+ * Author: Kira Toal
+ * Date: July 6, 2020
+ */ 
+public final class WriteAd {
+
+  public static void writeAd(Ad ad, String collection, String pathToServiceAccount, String databaseURL) throws Exception {
+    // Initialize app.
```

Comment:
Think their should be vertical whitespace between `writeAd` and the comment, but not 100% sure.

Reply:
Not on the first line of the method